app/components/blacklight/icons/other_component.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Blacklight
+  module Icons
+    class OtherComponent < Blacklight::Icons::IconComponent
```

Comment:
```suggestion
    class OtherResourceClassComponent < Blacklight::Icons::IconComponent
```
less confusing?

Reply:
We will have to add a mapping into the config if we do this.

Reply:
I guess it's already in the `Icons` namespace, so not as bad as it could be...

Reply:
We can do it, we would have to add the line 
other-resource: other here:
https://github.com/sul-dlss/earthworks/blob/bl8/config/settings.yml#L465-L466